src/build.ts

```diff
@@ -252,15 +254,28 @@ export async function build(
   }
 
   // Render pages!
+  const buildManifest: BuildManifest = {pages: []};
   for (const [sourceFile, {page, resolvers}] of pages) {
     const sourcePath = join(root, sourceFile);
     const outputPath = join(dirname(sourceFile), basename(sourceFile, ".md") + ".html");
     const path = join("/", dirname(sourceFile), basename(sourceFile, ".md"));
     effects.output.write(`${faint("render")} ${sourcePath} ${faint("→")} `);
     const html = await renderPage(page, {...config, path, resolvers});
     await effects.writeFile(outputPath, html);
+    const url =
+      outputPath === "index.html"
+        ? "/"
+        : basename(outputPath) === "index.html"
+        ? join("/", dirname(outputPath), "/")
+        : extname(outputPath) === ".html"
+        ? join("/", dirname(outputPath), basename(outputPath, ".html"))
+        : outputPath;
+    buildManifest.pages.push({url, title: page.title});
```

Comment:
Here `url` is actually a path rather than a URL. Can we call this `path` instead? (We use `page.path` elsewhere for a similar purpose.)

Reply:
In the future I'd like to also support listing the path of the source file and the generated HTML page, and I'm trying to make clear names for all there. What do you think about `urlPath`? That would suggest something like `sourcePath` and `distPath` in the future.

We don't need to build those feature now, but I don't want to close off the possibility. We likely won't be able to easily change this name in the future.

Or do you have other ideas?

Reply:
Ya, we have a similar need for disambiguation within Framework. I’ve been using (at least in my head) “serve path” for the path that is used to serve the page, so `foo/index.md` would be the “source path” and `foo/` would be the “serve path”. But generally I just refer to this as “path” as this is the public view of the page and thus in my mind more important than the source path…